README.md

```diff
@@ -6,104 +6,74 @@
 
 Eclipse Che is a next generation Eclipse IDE. This repository is licensed under the Eclipse Public License 2.0. Visit [Eclipse Che's Web site](https://eclipse.org/che) for feature information or the main [Che assembly repository](https://github.com/codenvy/che) for a description of all participating repositories.
 
-Che Dashboard
-
-==============
+# Eclipse Che Dashboard
 
 ## Requirements
 
-- Docker
+- Python `v2.7.x`(`v3.x.x`currently not supported)
+- Node.js `v8.x.x` or `v10.x.x`
+- yarn `v1.13.0` or higher
 
 ## Quick start
 
 ```sh
-cd che/dashboard
-mvn clean install
-```
-
-note: by default it will build dashboard using a docker image.
-If all required tools are installed locally, the native profile can be used instead of the docker build by following command:
-
-```sh
-$ mvn -Pnative clean install
+git clone https://github.com/eclipse/che-dashboard.git \
+  && cd che-dashboard \
+  && yarn \
+  && yarn serve
 ```
 
-Required tools for native build:
-
-- Python `v2.7.x`(`v3.x.x`currently not supported)
-- Node.js `v8.x.x` or `v9.x.x`
-- yarn `v1.13.0` or higher
-- gulp
-
 Installation instructions for Node.js can be found on the following [link](https://docs.npmjs.com/getting-started/installing-node).
 
 ## Running
 
-In order to run the project, the serve command is used
-
-```sh
-$ gulp serve
-```
-
-It will launch the server and then the project can be tested on http://localhost:3000
-
-By default it will use http://localhost:8080 as a remote server, so make sure that Che is running locally. More details about how to do that can be found on the following [link](https://github.com/eclipse/che/wiki/Development-Workflow#build-and-run---tomcat)  
-
-The argument `--server <url>` may allow to use another server. (url is for example http://my-server.com)
+In order to run the project, the `serve` command is used
 
 ```sh
-$ gulp serve:dist
+yarn serve
 ```
 
-This command will provide the 'minified/optimised' version of the application
+It will launch the dev server and then the project can be tested on http://localhost:3000.
 
-This is a good check for testing if the final rendering is OK
-
-## Tests
+By default https://che.openthift.io is used as a remote server, so make sure that Che is running locally.
```

Comment:
Why `so make sure that Che is running locally.` is che.openshift.io is used by default?

Reply:
You're right, providing the fix